/**:
  ros__parameters:
    lidar_list: ['lidar_0', 'lidar_1']
    lidar_0:
      port: "/dev/ttyUSB0"
      frame_id: "base_scan"
      baudrate: 230400
      version: 4
      topic_name: "scan_0"
      reverse: False
      warmup_time: 5  # seconds
    lidar_1:
      port: "/dev/ttyUSB1"
      frame_id: "base_scan"
      baudrate: 230400
      version: 4
      topic_name: "scan_1"
      reverse: False
      warmup_time: 5  # seconds
